doc/conf.py

```diff
@@ -95,6 +97,18 @@ def _check_deps():
   'pandas': ('http://pandas.pydata.org/pandas-docs/stable', None)
   }
 
+explicit_order_folders = ['../examples/pyplots',
+                          '../examples/statistics',
+                          '../examples/color',
+                          '../examples/pylab_examples',
+                          '../examples/showcase',
```

Comment:
Actually - I just realized that a better solution here might be to use the viewing data from google analytics to choose the ordering. That might be a kind of self-reinforcing kind of thing, but it's a good first start